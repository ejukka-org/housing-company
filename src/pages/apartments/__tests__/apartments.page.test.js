import React from 'react';
import ApartmentsPage from '../apartments.page';
import renderer from 'react-test-renderer';

test('Renders Apartments page', () => {
    const component = renderer.create(
        <ApartmentsPage />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
