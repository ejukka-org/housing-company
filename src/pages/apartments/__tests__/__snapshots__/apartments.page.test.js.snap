// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders About page 1`] = `
<div
  className="container"
>
  <div>
    <div
      className="carousel carousel-slider"
      style={
        Object {
          "width": "100%",
        }
      }
    >
      <button
        className="control-arrow control-prev"
        onClick={[Function]}
        type="button"
      />
      <div
        className="slider-wrapper axis-horizontal"
        style={Object {}}
      >
        <ul
          className="slider animated"
          onMouseDown={[Function]}
          onTouchEnd={[Function]}
          onTouchStart={[Function]}
          style={
            Object {
              "MozTransform": "translate3d(-100%,0,0)",
              "MozTransitionDuration": "350ms",
              "MsTransform": "translate3d(-100%,0,0)",
              "MsTransitionDuration": "350ms",
              "OTransform": "translate3d(-100%,0,0)",
              "OTransitionDuration": "350ms",
              "WebkitTransform": "translate3d(-100%,0,0)",
              "WebkitTransitionDuration": "350ms",
              "msTransform": "translate3d(-100%,0,0)",
              "msTransitionDuration": "350ms",
              "transform": "translate3d(-100%,0,0)",
              "transitionDuration": "350ms",
            }
          }
        >
          <li
            className="slide"
            onClick={[Function]}
          >
            <div>
              <img
                src="https://i.postimg.cc/YqCTRhHF/image3.jpg"
              />
              <p
                className="legend"
              >
                Erillistalo: Asunto A
              </p>
            </div>
          </li>
          <li
            className="slide selected"
            onClick={[Function]}
          >
            <div>
              <img
                src="https://i.postimg.cc/HsF3Mxsd/image1.jpg"
              />
              <p
                className="legend"
              >
                Paritalo: asunnot B2 ja B3
              </p>
            </div>
          </li>
          <li
            className="slide"
            onClick={[Function]}
          >
            <div>
              <img
                src="https://i.postimg.cc/158J46Sv/image2.jpg"
              />
              <p
                className="legend"
              >
                Erillistalo: Asunto A
              </p>
            </div>
          </li>
          <li
            className="slide"
            onClick={[Function]}
          >
            <div>
              <img
                src="https://i.postimg.cc/YqCTRhHF/image3.jpg"
              />
              <p
                className="legend"
              >
                Erillistalo: Asunto A
              </p>
            </div>
          </li>
          <li
            className="slide selected"
            onClick={[Function]}
          >
            <div>
              <img
                src="https://i.postimg.cc/HsF3Mxsd/image1.jpg"
              />
              <p
                className="legend"
              >
                Paritalo: asunnot B2 ja B3
              </p>
            </div>
          </li>
        </ul>
      </div>
      <button
        className="control-arrow control-next"
        onClick={[Function]}
        type="button"
      />
      <ul
        className="control-dots"
      >
        <li
          className="dot selected"
          onClick={[Function]}
          onKeyDown={[Function]}
          role="button"
          tabIndex={0}
          value={0}
        />
        <li
          className="dot"
          onClick={[Function]}
          onKeyDown={[Function]}
          role="button"
          tabIndex={0}
          value={1}
        />
        <li
          className="dot"
          onClick={[Function]}
          onKeyDown={[Function]}
          role="button"
          tabIndex={0}
          value={2}
        />
      </ul>
      <p
        className="carousel-status"
      >
        1 of 3
      </p>
    </div>
    <div
      className="carousel"
    >
      <div
        className="thumbs-wrapper axis-vertical"
      >
        <button
          className="control-arrow control-prev control-disabled"
          onClick={[Function]}
          type="button"
        />
        <ul
          className="thumbs animated"
          onMouseDown={[Function]}
          onTouchEnd={[Function]}
          onTouchStart={[Function]}
          style={
            Object {
              "MozTransform": "translate3d(0px,0,0)",
              "MozTransitionDuration": "350ms",
              "MsTransform": "translate3d(0px,0,0)",
              "MsTransitionDuration": "350ms",
              "OTransform": "translate3d(0px,0,0)",
              "OTransitionDuration": "350ms",
              "WebkitTransform": "translate3d(0px,0,0)",
              "WebkitTransitionDuration": "350ms",
              "msTransform": "translate3d(0px,0,0)",
              "msTransitionDuration": "350ms",
              "transform": "translate3d(0px,0,0)",
              "transitionDuration": "350ms",
            }
          }
        >
          <li
            className="thumb"
            onClick={[Function]}
            onKeyDown={[Function]}
            role="button"
            tabIndex={0}
          >
            <img
              onLoad={[Function]}
              src="https://i.postimg.cc/HsF3Mxsd/image1.jpg"
            />
          </li>
          <li
            className="thumb"
            onClick={[Function]}
            onKeyDown={[Function]}
            role="button"
            tabIndex={0}
          >
            <img
              src="https://i.postimg.cc/158J46Sv/image2.jpg"
            />
          </li>
          <li
            className="thumb"
            onClick={[Function]}
            onKeyDown={[Function]}
            role="button"
            tabIndex={0}
          >
            <img
              src="https://i.postimg.cc/YqCTRhHF/image3.jpg"
            />
          </li>
        </ul>
        <button
          className="control-arrow control-next control-disabled"
          onClick={[Function]}
          type="button"
        />
      </div>
    </div>
  </div>
</div>
`;
